src/middleware/index.js

```diff
@@ -10,7 +10,19 @@ const handle500 = ({ status = 500, name, message }, _req, res, next) => {
     res.status(status).json({ name, statusCode: status, message })
 }
 
+const checkEmailPasswordExist = (req, res, next) => {
+    const { email, password } = req.body
+    if (!email || !password) {
+        res.status(400).json({
+            message: "Please enter your email and password"
```

Comment:
If this was a UI error message, this'd be fine. But our api consumer is probably a dev and if they hit an error, a more useful error message would describe the problem more, I think. `Invalid email and/or password` maybe? Likewise, for the error message in `login.js`, `Invalid credentials` might be better than `That email or password is not correct.`. _I don't feel at all strongly about this_ but I do think it makes more sense.